react/features/app/actions.js

```diff
@@ -1,7 +1,11 @@
 /* @flow */
 
 import { setRoom } from '../base/conference';
-import { configWillLoad, loadConfigError, setConfig } from '../base/config';
+import {
+    configWillLoad,
+    loadConfigError,
+    setConfig
+} from '../base/config';
```

Comment:
Is that a coding style/formatting change or is it a leftover from earlier versions of this PR?